src/plasoscaffolder/bll/services/sqlite_plugin_helper.py

```diff
@@ -37,7 +44,7 @@ def formatter_file_path(path, plugin_name):
   return join(path, "plaso", "formatters", plugin_name + ".py")
```

Comment:
```
filename = "{0:s}.py".format(plugin_name)
return os.path.join(path, "plaso", "formatters", filename)
```

I opt to use the `os.path.join` here seeing a regular string join in Python is different.